composer.json

```diff
@@ -2,16 +2,16 @@
     "name": "spaggel/tooltip",
     "description": "Product attribute tooltips module for Magento 2",
     "require": {
-        "magento/framework": "~102.0",
-        "magento/module-backend": "~101.0",
-        "magento/module-catalog": "~103.0",
+        "magento/framework": "~102.0||~103.0",
+        "magento/module-backend": "~101.0||102.1",
+        "magento/module-catalog": "~103.0||104.0",
         "magento/module-store": "~101.0",
         "magento/module-ui": "~101.1",
-        "php": "~7.1.3||~7.2.0||~7.3.0"
+        "php": "~7.1.3||~7.2.0||~7.3.0||7.4.0"
     },
     "type": "magento2-module",
     "license": "MIT",
-    "version": "1.1.2",
+    "version": "1.1.3",
```

Comment:
@Spaggel, I would suggest removing the version from the `composer.json` as this leads to issues. E.g. see https://blog.packagist.com/tagged-a-new-release-for-composer-and-it-wont-show-up-on-packagist/. Do you agree?

Reply:
Agreed!